modelzip/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -eu
+
+#
+# this is a wrapper script to run the inference
+# Participants are expected to change this script to suit their model
+
+langs=$1
+batch_size=$2
+
+mydir=$(dirname "$0")
+mydir=$(realpath "$mydir")
+
+python -m modelzip.baseline $langs $batch_size -m $mydir
```

Comment:
would this work in the docker? I am afraid it won't as we just install requirements and the baseline.py is not even copied into the docker

Reply:
Thanks for catching this error. 
I had originally intended to copy baseline.py as run.py inside the model dir.  but that got changed and I assumed modelzip will be installed in the docker 
I will fix it before merging this PR.